src/AAVE/Card/CardsTable.jsx

```diff
@@ -0,0 +1,58 @@
+const { headers, datas, config } = props;
```

Comment:
`datas` is a typo

Reply:
This has been fixed in https://github.com/corndao/aave-v3-bos-app/pull/6/commits/fe4c77ca2c578189299c0e4d7dfe092940278255